concepticondata/conceptlists/Bowern-2012-348.tsv

```diff
@@ -0,0 +1,349 @@
+NUMBER	ID	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	COMMENTS
+1	Bowern-2012-348-1	1dl.excl	2636	WE TWO (EXCLUSIVE)	
+2	Bowern-2012-348-2	1dl.incl	2637	WE TWO (INCLUSIVE)	
+3	Bowern-2012-348-3	3dl	2635	THEY TWO	
+4	Bowern-2012-348-4	Bloodwood	3007	EUCALYPT	
+5	Bowern-2012-348-5	Blue-Tongue Lizard	632	LIZARD	mapping to LIZARD as it's the only Lacertilia in the list
+6	Bowern-2012-348-6	Brolga	1830	CRANE	mapping to CRANE as it's the only Gruidae in the list
```

Comment:
okay, we have to many special names here, so I'd in the end rather decide to leave them unmapped or to add them as new concepts and add concept relations, such as Blue tongue lizard is narrower than lizard. You should check with the file on conceptrelations where we list this.

Reply:
Ok, I see. What I tried to do was map to the closest match when there was a single entry (like BLUE-TONGUE LIZARD > LIZARD) and map one of the items to its closest match in case of multiple entries, adding the other ones.

I'll check the concept relations.

Reply:
Yes, I am afraid it is needed. It is already an almost uncontrolable task with more than 3000 concept sets. The more important that we try to stay as put as closely to strict definitions of concepts, which also revokes what I said earlier about the lizard. What I also do at times, and I find it justified, is just empty the cell and avoid the definition. We don't need to link everything, at times, we can allow to leave things unlinked, if we find they won't be important for cross-linguistic comparison.

Reply:
In this case it is actually easier: there are many concepts, for example, that really only make sense to Pama-Nyungan (and perhaps other Australian languages) and there is really no need to include in the large Concepticon-scope. I will take a new look at everything and maybe think of solutions to make this overlap identification easier.

Reply:
yes, you know, just leave them out. We have the NA class, which all concepts will be assigned to that we can't map, and we can later still get back an re-link. But it's good that you learned the general drill of adding a new concept, adding a concept-relation, and making the intro of a concept list. These are all crucial for working with lexibank (but also concepticon/cldf in general).